translate/pa3.1.md

```diff
@@ -0,0 +1,123 @@
+
+That is the concept of [Batch Processing System](https://en.wikipedia.org/wiki/Batch_processing), With the batch processing system, the administrator's hands can be freed. The key to the batch processing system is to have a background program. When a foreground program ends, the background program will automatically load a new foreground program to execute.
+
+Such a background program is actually an operating system. Yes, you heard it right, this background program that sounds like it does nothing is really an operating system! Speaking of operating systems, you may immediately think of Windows with several GB of installation packages. But in fact, the earliest operating system [GM-NAA I](https://en.wikipedia.org/wiki/GM-NAA_I/O) put into use in history was born in 1956, and one of its main tasks is to Go to "Automatically load new programs".
+
+#### What is an operating system? (It is recommended to think about it in the second trail)
+
+This is a big deal, and we encourage you to come back and revisit it after taking the operating system course.
+
+### [#](#The-simplest-operating-system) The simplest operating system
+
+Seeing is believing, let’s first show how simple the operating system can be. The operating system used in PA is called Nanos-lite, which is a tailored version of Nanjing University’s operating system Nanos. It is an operating system tailor-made for PA . By writing Nanos-lite code, you will realize how the operating system uses the mechanisms provided by the hardware (that is, ISA and AM) to support the running of the program, which is also in line with the ultimate goal of PA. As for the full version of Nanos , you will see its true colors in the operating system course next semester.
+
+We have prepared the framework code of Nanos-lite for everyone, which can be obtained by executing the following command:
+
+    cd ics2023
+    bash init.sh nanos-lite
+    
+
+Nanos-lite contains all modules used in subsequent PAs, but the specific functions of most modules have not been implemented. Since the functions of the hardware (NEMU) are gradually added, Nanos-lite must also cooperate with this process. You will use some macros in `nanos-lite/include/common.h` which are used to control the functions of Nanos-lite during the experiment. As the experimental progress progresses, we will gradually explain all the modules, and Nanos-lite will do more and more work. Therefore, when reading the code of Nanos-lite, you only need to care about the modules related to the current progress, and do not get entangled in the code that is not related to the current progress.
```

Comment:
```suggestion
Nanos-lite contains all modules used in subsequent PAs, but the functionality of most modules has not yet been implemented. Since the functionality of the hardware (NEMU) are gradually added, Nanos-lite must align with it. You will use some macros in `nanos-lite/include/common.h` which controls the feature set based on your progress. As the lab progresses, we will gradually explain all the modules, and Nanos-lite will do more and more work. Therefore, when reading the code of Nanos-lite, you only need to care about the modules related to the current progress, and do not get trapped in the code that is not related to the current progress.
```